.vscode/settings.json

```diff
@@ -18,5 +18,8 @@
     }
   },
   "typescript.tsdk": "node_modules/typescript/lib",
-  "vscode-vale.path": "${workspaceFolder}/node_modules/.bin/commercetools-vale"
+  "vscode-vale.path": "${workspaceFolder}/node_modules/.bin/commercetools-vale",
+  "workbench.colorCustomizations": {
+    "tab.unfocusedActiveBorder": "#fff0"
```

Comment:
😔

I guess we need to git ignore `settings.json` file again

Reply:
FF*. Every lovely morning day. I commit the bloody settings.json here.